<launch>
  <node pkg="husky_development" type="husky_driver" name="husky_vel" />
</launch>
